Library/Homebrew/commands.rb

```diff
@@ -0,0 +1,13 @@
+module Commands
+  def self.path(cmd)
```

Comment:
Just some nitpicking, but I thought the other implementation, `interal_command_path`, was much cleaner. I never liked the not quite switch statement/list of ifs this turned out to.